exasol_error_reporting_python/error_message_builder.py

```diff
@@ -0,0 +1,40 @@
+
+class ErrorMessageBuilder:
+    def __init__(self, error_code: str):
+        self._error_code = error_code
+        self._message_builder = []
+        self._mitigations = []
+
```

Comment:
parameter are missing

Reply:
Is there a need to parse a new format in python ? Parameters can be given to message using format function.

Reply:
We need it, because if you look at the other projects, you use the Parameters also have description. The name and the description get extracted by the crawler. This means we need to specify them separately

Reply:
error-reporting java and lua use different interface for 'parameters' method  (btw, csharp didn't handle it).
I'm following the way how java project handle it as follows:
`parameter(placeholder, value, description) `

Reply:
yes, that is fine